### PR TITLE
fix(tests): update travis.yml and update to dedicated API key

### PR DESCRIPTION
**Pull request by robinjoseph08**
### What

* Replace API key with one dedicated for Java 
* Update `.travis.yml` to fix issue with `/etc/hosts` relating to Travis https://github.com/travis-ci/travis-ci/issues/5227
* Swap the coveralls info to Lob's account
* Use the SVG badges

### CODE REVIEW

**Comment by javadev** [comment]
LGTM

**Comment by robinjoseph08** [comment]
Thanks for the help @javadev. I think I got the coveralls transferred over as well :+1: 

@mgartner

**Comment by javadev** [comment]
:+1:

**Comment by elnaz** [comment]
LGTM

File: lob-java-examples/src/main/java/com/lob/examples/AsyncJobExample.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
     // This example demonstrates launching multiple requests synchronously and transforming their results
     // once they come in. The only blocking call is the final get().
     public static void main(final String[] args) throws Exception {
-        final String apiKey = "test_0dc8d51e0acffcb1880e0f19c79b2f5b0cc"; // Replace this API key with your own.
+        final String apiKey = "test_7b1960d06d6dfe28d3862b98380b8b0dc93"; // Replace this API key with your own.
```
Comment (javadev) [inline lob-java-examples/src/main/java/com/lob/examples/AsyncJobExample.java]: Definition for aapiKey string may be moved to the class BaseExample as static protected constant.

**Comment by mgartner** [comment]
Looks fine to me, you might want to follow @javadev's advice so you only have to define `apiKey` once.

File: lob-java-examples/src/main/java/com/lob/examples/AsyncJobExample.java (hunk 1 above)
Comment (robinjoseph08) [inline lob-java-examples/src/main/java/com/lob/examples/AsyncJobExample.java]: Good call :+1: updated!

**Comment by mgartner** [comment]
@robinjoseph08 can we merge this now?

**Pull request merged**